### PR TITLE
Upgrade on-ramp-sdk to 1.0.10

### PR DESCRIPTION
**Pull request by wachunei**
**Description**

This PR upgrade on-ramp-sdk to 1.0.10 which includes a fix in the fees calculation for transak.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -106,13 +106,13 @@
     "oss-attribution-generator/**/debug": "^2.6.9"
   },
   "dependencies": {
-    "@consensys/on-ramp-sdk": "^1.0.9",
+    "@consensys/on-ramp-sdk": "^1.0.10",
     "@exodus/react-native-payments": "git+https://github.com/MetaMask/react-native-payments.git#dbc8cbbed570892d2fea5e3d183bf243e062c1e5",
     "@keystonehq/bc-ur-registry-eth": "^0.7.7",
     "@keystonehq/metamask-airgapped-keyring": "^0.3.0",
     "@keystonehq/ur-decoder": "^0.3.0",
-    "@metamask/controllers": "^30.0.2",
```
Comment (alaahd) [inline package.json:114]: @wachunei any idea why did we remove this package?

Comment (wachunei) [reply package.json:114]: We did not, it was automatically sorted by yarn in the package.json

File: package.json [hunk 2]
```diff
@@ -106,13 +106,13 @@
...
     "@metamask/contract-metadata": "^1.35.0",
+    "@metamask/controllers": "^30.0.2",
```
Comment (wachunei) [inline package.json:115]: @alaahd not removed ^

**Comment by wachunei** [comment]
superseded by #4818

**Pull request closed without merging**